packages/elements/src/components/app/app.component.ts

```diff
@@ -279,7 +279,7 @@ export class MutationTestReportAppComponent extends RealtimeElement {
     });
 
     const applySubscription = fromEvent(this.source, 'mutant-tested')
-      .pipe(debounceTime(UPDATE_CYCLE_TIME))
+      .pipe(auditTime(UPDATE_CYCLE_TIME))
```

Comment:
I used `auditTime` here, but it might be better to use `sampleTime` here:

![debouncetime-vs-throttletime-vs-audittime-vs-sampletime---rxjs-operators-compared(1)](https://github.com/stryker-mutator/mutation-testing-elements/assets/34238405/cd4eea9e-25bb-4948-bd85-f6479d70ccd5)

Reply:
That seems interesting. I don't feel strongly one way or the other. Let's create a separate PR for it if you think its better.